modules/special-rules/pages/shooting-at-war-engines.adoc

```diff
@@ -11,6 +11,9 @@ These categories reflect the fact that these units are so large and easy to hit.
 * There is no target category for 'Immobilised War Engine in the open' -- instead, you simply roll as many hit dice as the total Firepower that you direct at the unit.
 
 {blank}[TODO: Cross-reference the Firepower table, once we've added it. Also, we actually target detachments not units, and we need to clarify what happens when a War Engine detachment has a mix of Immobilised and non-Immobilised WE units.]
+{blank}[TODO: reply IJW, I think the easiest is to treat it as a detachment that's partially in cover - you can concentrate on the IMM units and get more dice, or potentially hit everyone and get fewer dice.]
+
+If a detachment of War Engine units has a mix of mobile and immobilised units, then you may choose to shoot at only the units that are immobilised, if you wish. If you do this then, when you allocate hits later in this procedure, you may not allocate hits to any of the units that still mobile.
```

Comment:
Good solution, makes sense to mirror the cover/not in cover situation. Just some tidy up to do, and a little rephrasing.

```suggestion
* If a target detachment contains both mobile and Immobilised War Engine units, then you may choose to shoot at only those that are Immobilised, if you wish.
If you do this then you may allocate hits to the Immobilised units **only**.
```

Reply:
For what it's worth, this was adapted directly from work-out-how-many-hit-dice-to-roll.adoc:

> If the target detachment has some units in cover and some in the open, then you may choose to shoot at only the units that are the open, if you wish. If you do this then, when you allocate hits later in this procedure, you may not allocate hits to any of the units that are in cover.

Reply:
Yes, but that text is in the topic that actually does form part of 'this procedure'. The edits we're making here aren't actually in a topic that is 'in this procedure'.